onebusaway-android/src/main/java/org/onebusaway/android/map/StopMapController.java

```diff
@@ -245,6 +245,11 @@ public void onSaveInstanceState(Bundle outState) {
     }
 
     @Override
+    public void onViewStateRestored(Bundle savedInstanceState) {
+
```

Comment:
Can you add a brief comment here on why we don't need to do anything in onViewStateRestored() for the StopMapController?